uvtools/plot.py

```diff
@@ -471,6 +472,428 @@ def omni_view_gif(filenames, name='omni_movie.gif'):
+    )
+    if draw_colorbar:
+        if return_value is ax:
+            fig = ax.get_figure()
+
+        extend = "neither" if dynamic_range is None else "min"
+        cbar = fig.colorbar(mappable=scalar_map, ax=ax, extend=extend)
+        cbar.set_label(cbar_label, fontsize=fontsize)
+
+    return return_value
+
+def fourier_transform_waterfalls(
+    data,
+    antpairpol=None,
+    freqs=None,
+    times=None,
+    lsts=None,
+    time_or_lst="lst",
+    mode="log",
+    figsize=(14,10),
+    dpi=100,
+    aspect="auto",
+    fontsize=None,
+    cmap="best",
+    dynamic_range=None,
+    plot_limits=None,
+    freq_taper=None,
+    freq_taper_kwargs=None,
+    time_taper=None,
+    time_taper_kwargs=None,
+):
+    """
+    Plot a 2x2 grid of waterfalls showing all possible Fourier transforms.
+
+    Moving clockwise from the top-left, the plots are as follows:
+        time vs frequency
+        fringe-rate vs frequency
+        fringe-rate vs delay
+        time vs delay
+
+    Time units used for plotting are either JD or LST hours.
+    Frequency units used for plotting are MHz.
+    Fringe-rate units used for plotting are mHz.
+    Delay units used for plotting are ns.
+    
+    Note that these units may not correspond exactly to the units of the related
+    parameters passed. The parameter units for ``freqs`` and ``times`` or ``lsts``
+    are chosen to match the units typically used for recording metadata in
+    :class:`pyuvdata.UVData` objects.
+
+    Parameters
+    ----------
+    data: array-like of complex, or :class:`pyuvdata.UVData` instance
+        Object containing visibility data. If an array is passed, then ``freqs``
+        and either ``times`` or ``lsts`` must be provided, and the array must
+        have shape (``lsts.size``, ``freqs.size``). Otherwise, an ``antpairpol``
+        key must be provided.
+    antpairpol: tuple
+        (ant1, ant2, pol) tuple specifying the baseline and polarization to
+        pull data for if ``data`` is a :class:`pyuvdata.UVData` instance. Ignored
+        if ``data`` is an array-like object.
+    freqs: array-like of float
+        Frequencies corresponding to the observed data, in Hz. Required if ``data``
+        is an array-like object; ignored otherwise.
+    times: array-like of float
+        Observation times, in JD. Required if ``data`` is an array-like object and
+        ``lsts`` is not provided.
+    lsts: array-like of float
+        Observed LSTs, in radians. Required if ``data`` is an array-like object
+        and ``times`` is not provided.
+    time_or_lst: str, optional
+        Either "time" or "lst". Used to specify whether the time axis should be
+        in JD or LST. If ``data`` is an array-like object and only one of ``times``
+        or ``lsts`` is provided, then this parameter is ignored.
+    mode: str, optional
+        Plotting mode to use; must be a mode accepted by :func:`data_mode`. Default
```

Comment:
What is :func:data_mode? How do I get a list of options?

Reply:
Ah, that type of docstring formatting is intended to produce a link to the function in the compiled `rst` version of the documentation (so on the readthedocs). `data_mode` is the first function defined in the `plot` module.